### DISCUSSION TITLE
[Unstable] Separatist outfits found in coalition outposts

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14478] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The new separatist outfits are sold by coalition merchants. It doesn't really make sense for a coalition merchant to be selling separatist clothing.
Additionally, they can be found in outpost crew cabinets at the exact same rate as the regular outfits, and while it's not impossible for hiding separatists to be storing their clothing while in a coalition station, it doesn't really make sense for them to be just as common as the other outfits.

### Reproduction steps

1. Start campaign
2. Check coalition merchant, observe that they are selling separatist gear
3. Look around in cabinets, observe that separatists are seemingly everywhere

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next Unstable build.
